app/moves/middleware/set-filter.moves.test.js

```diff
@@ -159,18 +174,61 @@ describe('Moves middleware', function () {
           const filter = [
             {
               label: 'statuses::pending',
-              href: '/moves/week/2010-09-07/123?foo=bar',
+              href: '/moves/week/2010-09-07/123?foo=bar&status=pending',
               value: 4,
+              active: false,
             },
             {
               label: 'statuses::approved',
-              href: '/moves/week/2010-09-07/123?foo=bar',
+              href: '/moves/week/2010-09-07/123?foo=bar&status=approved',
               value: 4,
+              active: false,
             },
             {
               label: 'statuses::rejected',
-              href: '/moves/week/2010-09-07/123?foo=bar',
+              href: '/moves/week/2010-09-07/123?foo=bar&status=rejected',
               value: 4,
+              active: false,
+            },
+          ]
+          expect(req.filter).to.deep.equal(filter)
+          expect(req.filterBodyKey).to.deep.equal(filter)
+        })
+
+        it('calls next', function () {
+          expect(next).to.have.been.calledWithExactly()
+        })
+      })
+
+      context('with current status active', function () {
+        beforeEach(async function () {
+          req.query = {
+            foo: 'bar',
+          }
+          req.body[mockBodyKey].status = 'pending'
+
+          await middleware(mockConfig, mockBodyKey)(req, res, next)
+        })
+
+        it('should set current status to active', function () {
```

Comment:
> current status to active

This doesn't mean that `status` is set to `active`, it means that the `status` has one of the active values (e.g. `pending` `approved` `rejected`

:+1: